plugins/k8saudit/rules/k8s_audit_rules.yaml

```diff
@@ -741,3 +741,31 @@
   priority: WARNING
   source: k8s_audit
   tags: [k8s]
+
+- list: excessively_capable_list
+  items: [
+    CAP_SYS_ADMIN,
+    CAP_SYS_MODULE,
+    CAP_SYS_RAWIO,
+    CAP_SYS_PTRACE,
+    CAP_SYS_BOOT,
+    CAP_SYSLOG,
+    CAP_DAC_READ_SEARCH,
+    CAP_NET_ADMIN,
+    CAP_BPF
+    ]
+
+- list: falco_capable_images
+  items: []
+
+- rule: Launch Excessively Capable Pod
+  desc: >
+    Detect an attempt to start a pod with an excessively capable
+  condition: >
+    kevt and pod and kcreate and ka.req.pod.containers.add_capabilities intersects (excessively_capable_list)
+    and not ka.req.pod.containers.image.repository in (falco_privileged_images)
```

Comment:
we need cover more exceptions here, `kube-proxy`, kube-dns` and any other CSI/CNI  services more or less use extra capabilities.

Reply:
I see. But I'm now testing only in GKE, and there are only `NET_BIND_SERVICE` `DAC_OVERRIDE` `SETGID` caps except privileged one.
Could you tell me or create suggestion about the pods we should ignore in other Kubernetes clusters like on AKS, EKS?

Reply:
cc @alacuku